zcbor/zcbor.py

```diff
@@ -47,6 +47,9 @@ def is_relative_to(path1, path2):
 if Path(__file__).name in sys.argv[0]:
     # Running the script directly in the repo.
     c_code_root = P_REPO_ROOT
+elif any((match(r"zcbor-.*\.egg", p) for p in P_SCRIPT.parts)):
```

Comment:
should this be here?

Reply:
Yes, Added a note in the commit message.